modules/heat_conduction/include/bcs/GapHeatTransfer.h

```diff
@@ -77,5 +84,17 @@ class GapHeatTransfer : public IntegratedBC
 
   Point & _p1;
   Point & _p2;
-};
 
+  /// The current \p PenetratationInfo
+  const PenetrationInfo * _pinfo;
+
+  /// The phi values on the slave side
+  const std::vector<std::vector<Real>> * _slave_side_phi;
+
+  /// The slave side element (this really is a *side* element, e.g. it has
+  /// dimension: mesh_dimension - 1)
+  const Elem * _slave_side;
+
+  /// The slave side shape index
+  unsigned int _slave_j;
```

Comment:
You probably should initialize each of these new POD class members

Reply:
Done